### PR TITLE
Refactor non-code-path resources from FS => FX

### PR DESCRIPTION
**Pull request by opti-jnguyen**
**Summary**
- Updated readme file and matched its format with the provided sample readme file.
- Removed all occurrences of full stack with Optimizely Feature Experimentation
- Refactor required due to rename of Full Stack product to Feature Experimentation.

**Issues**
[FSSDK-8960](https://jira.sso.episerver.net/browse/FSSDK-8960)

### CODE REVIEW

**Comment by opti-jnguyen** [comment]
@russell-loube-optimizely - ready for review.

Unrelated, but also noticed there's no review requirements for this repository.

File: README.md [hunk 1]
```diff
@@ -1,73 +1,78 @@
...
 
----
+Optimizely Rollouts is [free feature flags](https://www.optimizely.com/free-feature-flagging/) for development teams. You can easily roll out and roll back features in any application without code deploys, mitigating risk for every feature on your roadmap.
 
-## Optimizely + AWS Lambda@Edge Starter Kit
+## Get Started
 
-> Starter Kit for running Optimizely experiments on [AWS Lambda@Edge Functions](https://aws.amazon.com/lambda/edge/) using AWS Lambda and AWS Cloudfront.
+Refer to the [Optimizely AWS Lambda@Edge Starter Kit documentation page](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/aws-lambda-at-edge) for detailed instructions about using this starter kit.
```
Comment (russell-loube-optimizely) [inline README.md]: Nit:
```suggestion
Refer to the [Optimizely AWS Lambda@Edge Starter Kit documentation](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/aws-lambda-at-edge) for detailed instructions about using this starter kit.
```

File: README.md [hunk 2]
```diff
@@ -1,73 +1,78 @@
...
+
+> Note: This starter kit in particular makes use of the "Lite" version of our Javascript SDK for Node.js which explicitly excludes the datafile manager and event processor features for better performance. As a result, it is expected that you will provide the datafile manually to the Optimizely SDK either through a local file reference or by using the provided platform-specific `getDatafile()` helper to load in your Optimizely project's datafile.
 
-   - Set up and configure AWS Lambda@Edge. For a tutorial on how to setup a basic AWS Lambda@Edge environment, [click here](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-edge-how-it-works-tutorial.html).
-   - Have an Optimizely account. If you do not have an account, you can [register for a free account](https://www.optimizely.com/products/intelligence/full-stack-experimentation/).
+### Initializing and Using Optimizely
```
Comment (russell-loube-optimizely) [inline README.md]:
```suggestion
### Initialization
```

File: README.md [hunk 3]
```diff
@@ -76,20 +81,78 @@ You will need to complete the following prerequisites to use this template:
...
 
 - [Lambda - Lambda@Edge offical documentation](https://docs.aws.amazon.com/lambda/latest/dg/lambda-edge.html)
 - [CloudFront - Lambda@Edge documentation](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-at-the-edge.html)
 - [CloudFront - Lambda@Edge Get Started](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-edge-how-it-works.html)
 - [Example Lambda Functions](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-examples.html)
 - [Article: Lambda@Edge Gotchas and Tips](https://medium.com/@mnylen/lambda-edge-gotchas-and-tips-93083f8b4152)
-- [AWS Lambda@Edge with Optimizely documentation](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/aws-lambda-at-edge)
+- [Optimizely AWS Lambda@Edge Starter Kit Documentation](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/aws-lambda-at-edge)
+
+## Contributing
```
Comment (russell-loube-optimizely) [inline README.md]:
```suggestion
## SDK Development

### Contributing
```

File: README.md [hunk 4]
```diff
@@ -76,20 +81,78 @@ You will need to complete the following prerequisites to use this template:
...
+If you have further questions, comments, concerns, or contributions, feel free to reach out via GitHub Issues! Please see [CONTRIBUTING](CONTRIBUTING.md).
```
Comment (russell-loube-optimizely) [inline README.md]:
```suggestion
Please see [CONTRIBUTING](CONTRIBUTING.md).
```

File: README.md [hunk 5]
```diff
@@ -76,20 +81,78 @@ You will need to complete the following prerequisites to use this template:
...
+
+- Ruby - https://github.com/optimizely/ruby-sdk
+
+- Swift - https://github.com/optimizely/swift-sdk
+
+### Other Optimizely Edge Starter Kits
+
+- Akamai EdgeWorkers - https://github.com/optimizely/akamai-edgeworker-starter-kit
+
+- AWS Lambda@Edge - https://github.com/optimizely/aws-lambda-at-edge-starter-kit
```
Comment (russell-loube-optimizely) [inline README.md]:
```suggestion
```

**Review by andrewleap-optimizely — APPROVED** [review]
lgtm

**Pull request merged**